scripts/us_census/pep_srh/process.py

```diff
@@ -0,0 +1,1047 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This module creates CSV files used for importing data into DC.
```

Comment:
Please put Comment section in three double quotes  ( " " ") as per Python Style guide

Reply:
Changed Doc string comments from single quotes to double quotes.